index.js

```diff
@@ -337,12 +370,12 @@ module.exports = function StateProvider(makeRenderer, rootElement, stateRouterOp
 	stateProviderEmitter.stateIsActive = (stateName = null, parameters = null) => {
 		const currentState = lastCompletelyLoadedState.get()
 		const stateNameMatches = currentState.name === stateName
-			|| currentState.name.indexOf(stateName + `.`) === 0
+			|| currentState.name.indexOf(`${stateName }.`) === 0
 			|| stateName === null
 		const parametersWereNotPassedIn = !parameters
 
 		return stateNameMatches
-			&& (parametersWereNotPassedIn || Object.keys(parameters).every(key => parameters[key] + `` === currentState.parameters[key]))
+			&& (parametersWereNotPassedIn || Object.keys(parameters).every(key => `${parameters[key] }` === currentState.parameters[key]))
```

Comment:
```suggestion
			&& (parametersWereNotPassedIn || Object.keys(parameters).every(key => `${parameters[key]}` === currentState.parameters[key]))
```

This space wasn't here before, was it?  I'm surprised this wouldn't break a test

Reply:
Geez I thought I disabled that linting rule while working on this, but I guess not. I can just revert it as well.

Reply:
ah you're right, I misread it 😂 

I suppose it's just as valid to suggest that there be whitespace on both sides of the expression, rather than just on the right side
```suggestion
			&& (parametersWereNotPassedIn || Object.keys(parameters).every(key => `${ parameters[key] }` === currentState.parameters[key]))
```